pom.xml

```diff
@@ -142,6 +142,60 @@
             <artifactId>jakarta.ws.rs-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
Hi Mark, can you define the version of the dependency in the dependencyManagement section of the pom and only define the usage in this part.